docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -124,4 +123,75 @@ An {oldetection} configuration object has the following properties:
 `feature_influence_threshold`:: 
   (double) The minimum {olscore} that a document needs to have in order to 
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+
+[discrete]
+[[regression-resources]]
+==== {regression-cap} configuration objects
+
+A `{regression}` configuration object has the following properties:
+
+`dependent_variable`::
+  (Required, string) Defines which field of the {dataframe} is to be predicted. 
+  The data type of the field must be numeric.
+
+`eta`::
+  (Optional, double) The shrinkage applied to the weights. 
+
+`feature_bag_fraction`::
+  (Optional, double) Defines the fraction of features which will be used when 
+  selecting a random bag for each candidate split. 
+
+`gamma`::
+  (Optional, double) A regularisation parameter associated with tree training. 
+  Defines a linear penalty associated with tree size. A higher value means 
+  training will prefer smaller trees. It aims to prevent overfitting.
+  
+`lambda`::
+  (Optional, double) A regularisation parameters associated with tree training. 
+  It is the L2 regularisation term which applies to leaf weights of the 
+  individual trees in the forest. It aims to prevent overfitting.
+  
+`maximum_number_trees`::
+  (Optional, integer) Defines the maximum number of trees the forest will be 
+  allowed to contain. The maximum value is 2000.
+  
+`prediction_field_name`::
+  (Optional, string) Defines the name of the prediction field in the results. 
+  Defaults to `<dependent_variable>_prediction`.
+  
+`training_percent`::
+  (Optional, integer) Defines what percentage of data frame rows that are 
+  eligible for training are actually going to be used for training. Defaults to 
+  `100`.
+
+
+[source,js]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/wine_quality_white
+{
+  "source": {
+    "index": "wine_quality_white"
+  },
+  "dest": {
+    "index": "wine_quality_white_regression"
+  },
+  "analysis": 
+    {
+      "regression": {
+        "dependent_variable": "quality"
+      }
+    }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[setup:setup_wine_quality_white]
+
+
+[discrete]
+[[regression-resources-response]]
+==== {regression-cap} response object
+
+`is_training`::
```

Comment:
I don't think this is actually in the response body. I think it is a field that is written to the resulting document after the analyses is complete.